modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/openapi/v1_0/ObjectEntryOpenAPIResourceImpl.java

```diff
@@ -362,6 +362,17 @@ private OpenAPISchemaFilter _getOpenAPISchemaFilter(
 							objectField),
 						String.class.getSimpleName()) {
 
+						{
+							setRequired(objectField.isRequired());
+						}
+					});
+
+				dtoProperties.add(
+					new DTOProperty(
+						Collections.singletonMap("x-parent-map", "properties"),
+						objectRelationship.getName() + "ERC",
+						String.class.getSimpleName()) {
+
 						{
 							setRequired(objectField.isRequired());
```

Comment:
I am not sure if the field is required... I think should be read only.

I have created two object definitions, Student and University and created a relationship (one to many) between them. Then, I have created a university and I got the ERC. Then, I try to create a Student and send this field with the ERC of the university:

```curl -X POST -u test@liferay.com:test http://localhost:8080/o/c/students -H 'Content-type: application/json' -d '{"studentName": "student6", "universityStudentsERC": "4cb30281-d2cf-b77f-3e95-e576b9d67803"}'```

Response:

```
{
 [...]
  "externalReferenceCode" : "f1adb466-3e83-7ab0-213b-7fd8759d5c3f",
  "id" : 33806,
  [...]
  "r_universityStudents_c_universityId" : 0,
  "universityStudentsERC" : "",
  "studentName" : "student6",
  "r_universityStudents_c_universityERC" : ""
}
```

The result is the student is NOT related to the university. If I use the `r_universityStudents_c_universityERC` field instead, it works. So I would say the best approach is to set this field as read only and not mandatory. What do you think?

cc/ @4lejandrito